### PR TITLE
Tasks page

### PR DESCRIPTION
**Pull request by RakshaPawar108**
- Made a Create Task modal
- Styled task modal
- Added it to Tasks page
- Gave logic to open and close task modal

**This is the second PR of Navbar and Footer branch which is being made because I added the task page related commits in this branch by mistake**

**Deploy Preview Link: https://deploy-preview-3--task-todo-ro.netlify.app/**

### CODE REVIEW

File: src/components/Tasks/TaskModal/TaskModal.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.input-container input,
+.input-container textarea {
+  font-size: 1.5rem;
+}
+
```
Comment (gadgetfather) [inline src/components/Tasks/TaskModal/TaskModal.css:45]: also give `resize: none` so it won't be resizable.

File: src/pages/Tasks/Tasks.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.tasks-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  height: vh;
```
Comment (gadgetfather) [inline src/pages/Tasks/Tasks.css]: here you forgot to give value.

**Review by gadgetfather — APPROVED** [review]
Modal is working perfectly. 
Really liked the textarea placeholder animation.
Great job @RakshaPawar108

**Review by mohammed-sas — APPROVED** [review]
open and close task modal is working fine
you could use custom hooks for toggling purpose
code is clean and readable

**Pull request merged**